### PR TITLE
Add Chinese documents

### LINKED ISSUE #9: Chinese version of deploying applications to multiple clusters
翻译源: https://github.com/clusternet/website/blob/main/content/en/docs/tutorials/deploy-to-multiple-clusters.md
翻译后: `content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md`

### PR DESCRIPTION
**Pull request by MentosL**
Add Chinese documents

Fixes #9

### CODE REVIEW

File: content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+> `Deployer`相关特性应该由`clusternet-hub`启用。
+
+- [定义你的应用](/docs/tutorials/deploy-to-multiple-clusters/#defining-your-applications)
+- [重新设置相关配置](/docs/tutorials/deploy-to-multiple-clusters/#setting-overrides)
+- [应用你的应用程序](/docs/tutorials/deploy-to-multiple-clusters/#applying-your-applications)
+- [检查状态](/docs/tutorials/deploy-to-multiple-clusters/#checking-status)
+
+## 定义你的应用
+
+首先，，我们先看下引用示例应用的定义，在`Subscription` "app-demo" 下面定义了目标子集群
```
Comment (dixudx) [inline content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md:23]: remove redundant commas.

**Comment by MentosL** [comment]
> @MentosL Please sign off your commits.
> 
> ```shell
> git commit -s --amend
> git push -f origin main
> ```

ok ,already exec it

File: content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md (hunk 1 above)
Comment (dixudx) [reply content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md:23]: s/定义了目标子集群分发到/定义了要分发的目标子集群

File: content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+分发到，以及要部署的资源。
+
+```examples/applications/subscription.yaml
+apiVersion: apps.clusternet.io/v1alpha1
+kind: Subscription
+metadata:
+  name: app-demo
+  namespace: default
+spec:
+  subscribers: # defines the clusters to be distributed to
```
Comment (dixudx) [inline content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md:33]: We can translate the comment here.

File: content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    - clusterAffinity:
+        matchLabels:
+          clusters.clusternet.io/cluster-id: dc91021d-2361-4f6d-a404-7c33b9e01118 # PLEASE UPDATE THIS CLUSTER-ID TO YOURS!!!
+  feeds: # defines all the resources to be deployed with
```
Comment (dixudx) [inline content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md:37]: Please help translate the comments in this yaml file. Thanks.

File: content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+    - apiVersion: apps/v1
+      kind: Service
+      name: my-nginx-svc
+      namespace: foo
+    - apiVersion: apps/v1
+      kind: Deployment
+      name: my-nginx
+      namespace: foo
+```
+在应用`Subscription`前，请使用你的clusterID修改部分值。
```
Comment (dixudx) [inline content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md:54]: s/使用你的clusterID修改部分值/将修改clusterID

File: content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+ [examples/applications/subscription.yaml](https://github.com/clusternet/clusternet/blob/main/examples/applications/subscription.yaml)
+
+
+> :bulb: :bulb:
+> 如果要从私有 helm 仓库安装 helm chart，请参考[this example](https://github.com/clusternet/clusternet/blob/main/deploy/templates/helm-chart-private-repo.yaml)设置有效的 `chartPullSecret`。
```
Comment (dixudx) [inline content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md:59]: s/this example/这个例子

File: content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+
+## 重新设置相关配置
```
Comment (dixudx) [inline content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md:61]: s/重新设置相关配置/设置覆盖值

**Comment by MentosL** [comment]
OK, the above has been processed

File: content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+
+`Clusternet` 还提供了***基于两阶段优先级的***覆盖策略。 你可以定义有优先级的命名空间范围的`本地化`和集群范围的`全局化`（范围从0到1000，默认为为 500），
```
Comment (dixudx) [inline content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md:63]: s/本地化/Localization

s/全局化/Globalization

`Localization` and `Globalization` are CRD names. Don't translate them.

File: content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+其中较低的数字被认为是较低的优先级。这些全局化和本地化将被应用按优先级从低到高的顺序。这意味着较低的`全局化`中的覆盖值将被那些覆盖在更高的`全局化`中。首先是全局化，然后是本地化。
```
Comment (dixudx) [inline content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md:64]: > 全局化
> 本地化

Same as above

File: content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+- 默认覆盖策略`ApplyLater`只会在下次更新时应用覆盖匹配的对象（包括更新在 `Subscription`、`HelmChart` 等）或新创建的对象。
+
+
+在应用这些本地化之前，请
+修改[examples/applications/localization.yaml](https://github.com/clusternet/clusternet/blob/main/examples/applications/localization.yaml)
+使用您的`ManagedCluster`命名空间，例如`clusternet-5l82l`。
+
+## 应用你的应用程序
+
+安装 kubectl 插件 [kubectl-clusternet](/docs/kubectl-clusternet/) 后，您可以运行 下面的命令将此应用程序分发到子集群。
```
Comment (dixudx) [inline content/zh-cn/docs/tutorials/deploy-to-multiple-clusters.md:82]: > 运行 下面

运行下面

**Comment by MentosL** [comment]
OK, the above has been processed again

**Comment by dixudx** [comment]
@MentosL Please remember to sign off every commit.

**Comment by dixudx** [comment]
@MentosL Thanks for your contribution! Looking forwards to make more contributions.

**Comment by MentosL** [comment]
> @MentosL Thanks for your contribution! Looking forwards to make more contributions.

ok, thanks for the guide

**Pull request merged**